index.html

```diff
@@ -0,0 +1,181 @@
+                <img src="images/Snapshoot4.png" alt="snapshoot4">
+            </div>
+            <h1>Multi-Post
+                Stories</h1>
+            <div class="text-info">
+                <ul>
+                <li class="client-name">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            </div>
+            <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+    </section>
+                                 <!-- start About me -->
+    <section class="about-me">
+        
+        <div class="about-me-text">
+            <h2>About
+                 Myself</h2> 
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature
+                     or website Look through some of my work and experience! If you like what 
+                     you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+        </div>
+        <div class="hero-socialmedia-icons">
+            <ul>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/linkdin.png" alt="linkdin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+        </ul>
+        </div>
+        <button class="see-proj">Get My Resume</button>
+        
+        <!--skills Section in about-me -->
+    <div class="skills">
+        <div class="langguages">
+            <ul>
+                <li> <h3>langguages</h3></li>
+                <li><img src="images/down-icon.svg" alt="down-icon"></li>
+            </ul>
+        </div>
+        <div class="lang-submenu">
+            <ul>
+             <li><img src="images/js-icon.png" alt="js-icon"> JavaScript </li>
+             <li><img src="images/html5-icon.png" alt="html5-icon"> Html </li>
+             <li><img src="images/css3-icon.png" alt="css3-icon"> Css </li>
+            </ul>
+
+        </div>
+        <ul>
+            <li><h3>Framworks</h3></li>
+            <li><img src="images/right-icon.svg" alt="right-icon"></li>
+        </ul>
+        <img src="images/line.png" alt="h-line">
+        <ul>
+            <li> <h3>Skills</h3></li>
+            <li><img src="images/right-icon.svg" alt="right-icon"></li>
+        </ul>
+        <img src="images/line.png" alt="h-line">
+    </div> <!--end of Skills part-->
```

Comment:
- [ ] This section should be a list of lists, with each parent (Languages, Frameworks, Skills) being a list element of an unordered list, and being on their own another unordered list (with their list items being for example Javascript, HTML, CSS).

Reply:
I can't understand this! as I know I did it ("Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items".)